app/design/adminhtml/default/openmage/template/resetforgottenpassword.phtml

```diff
@@ -0,0 +1,96 @@
+    </head>
+    <body id="page-login">
+        <div class="login-container">
+            <div class="login-box">
+                <form method="post" action="<?php echo $this->getUrl('*/*/resetpasswordpost', array('_query' => array('id' => $userId, 'token' => $resetPasswordLinkToken))); ?>" id="reset-password-form">
+                    <fieldset class="login-form">
+                        <input name="form_key" type="hidden" value="<?php echo $this->getFormKey(); ?>" />
+                        <h2><?php echo Mage::helper('adminhtml')->__('Reset a Password'); ?></h2>
+                        <div id="messages">
+                            <?php echo $this->getMessagesBlock()->toHtml(); ?>
+                        </div>
+                        <div class="input-box f-left">
+                            <label for="password"><em class="required">*</em> <?php echo $this->__('New Password'); ?></label>
+                            <br />
+                            <!-- This is a dummy hidden field to trick firefox from auto filling the password -->
+                            <input type="password" class="input-text no-display" name="dummy" id="dummy" />
+                            <input type="password" class="input-text required-entry validate-admin-password" name="password" id="password" autocomplete="new-password"/>
+                        </div>
+                        <div class="input-box f-right">
+                            <label for="confirmation"><em class="required">*</em> <?php echo $this->__('Confirm New Password'); ?></label>
+                            <br />
+                            <!-- This is a dummy hidden field to trick firefox from auto filling the password -->
+                            <input type="password" class="input-text no-display" name="dummy" id="dummy" />
+                            <input type="password" class="input-text required-entry validate-cpassword" name="confirmation" id="confirmation" autocomplete="new-password"/>
+                        </div>
+                        <div class="clear"></div>
+                        <div class="form-buttons">
+                            <a class="left" href="<?php echo $this->getUrl('adminhtml', array('_nosecret' => true)) ?>">&laquo; <?php echo Mage::helper('adminhtml')->__('Back to Login'); ?></a>
+                            <button type="submit" title="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Reset Password')); ?>" class="forgot-password"><span><span><span><?php echo Mage::helper('adminhtml')->__('Reset Password'); ?></span></span></span></button>
+                        </div>
+                    </fieldset>
+                    <p class="legal"><?php echo Mage::helper('adminhtml')->__('Magento is a trademark of Magento Inc. Copyright &copy; %s Magento Inc.', date('Y')); ?></p>
```

Comment:
We removed all Magento logos and the word Magento so we should be able to also remove this legal notice.